README.rst

```diff
@@ -14,18 +14,78 @@ For full details see http://figure.openmicroscopy.org
 
 
 Requirements
-------------
+============
 
-* OMERO 4.4.x or OMERO 5.x
+* OMERO 5.2.6 or newer.
 
 Installation
-------------
+============
+
+Install OMERO.web.
+
+This app installs into the OMERO.web framework.
```

Comment:
I don't know about Dockerfile. The existing text "installs into the OMERO.web framework" sounds like omero_figure is being placed into an existing OMERO.web directory. But when you do ``` pip install omero_figure ``` this is completely independent of OMERO.web. It's a minor comment - maybe I'm being too picky.